src/main/java/minicraft/screen/entry/TextAreaEntry.java

```diff
@@ -0,0 +1,255 @@
+package minicraft.screen.entry;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import minicraft.core.io.InputHandler;
+import minicraft.gfx.Color;
+import minicraft.gfx.Font;
+import minicraft.gfx.FontStyle;
+import minicraft.gfx.Screen;
+import minicraft.util.ClipboardSystem;
+
+public class TextAreaEntry extends InputEntry {
+	private String prompt;
+	private String regex;
+	private int maxLength;
```

Comment:
All of these three are redundant.

Reply:
Since it is not visible, so I change them to protected now.